app/screens/home/RouteList.tsx

```diff
@@ -270,32 +281,32 @@ const RouteList = ({
           { condition: true, label: "model.route.rewards", value: `${formatAmount(route.rewardVolume)} EUR` },
         ]
       : // buy route
-      [
-        { condition: true, label: "model.route.type", value: t(`model.route.${route.type.toLowerCase()}`) },
-        { condition: route.type === BuyType.WALLET, label: "model.route.asset", value: route.asset?.name },
-        {
-          condition: route.type === BuyType.STAKING,
-          label: "model.route.staking",
-          value: `${t("model.route." + route.staking?.rewardType.toLowerCase())} - ${t(
-            "model.route." + route.staking?.paybackType.toLowerCase()
-          )}`,
-        },
-        { condition: true, label: "model.route.iban", value: route.iban },
-        {
-          condition: true,
-          label: "model.route.bank_usage",
-          value: route.bankUsage,
-          icon: "content-copy",
-          onPress: () => ClipboardService.copy(route.bankUsage),
-        },
-        {
-          condition: true,
-          label: "model.route.fee",
-          value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
-        },
-        { condition: true, label: "model.route.volume", value: `${formatAmount(route.volume)} €` },
-        { condition: true, label: "model.route.annual_volume", value: `${formatAmount(route.annualVolume)} €` },
-      ];
+        [
+          { condition: true, label: "model.route.type", value: t(`model.route.${route.type.toLowerCase()}`) },
+          { condition: route.type === BuyType.WALLET, label: "model.route.asset", value: route.asset?.name },
+          {
+            condition: route.type === BuyType.STAKING,
+            label: "model.route.staking",
+            value: `${t("model.route." + route.staking?.rewardType.toLowerCase())} - ${t(
+              "model.route." + route.staking?.paybackType.toLowerCase()
+            )}`,
+          },
+          { condition: true, label: "model.route.iban", value: route.iban },
+          {
+            condition: true,
+            label: "model.route.bank_usage",
+            value: route.bankUsage,
+            icon: "content-copy",
+            onPress: () => ClipboardService.copy(route.bankUsage),
+          },
+          {
+            condition: true,
+            label: "model.route.fee",
+            value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
+          },
+          { condition: true, label: "model.route.volume", value: `${formatAmount(route.volume)} €` },
+          { condition: true, label: "model.route.annual_volume", value: `${formatAmount(route.annualVolume)} €` },
+        ];
```

Comment:
I guess I need to check why my formatting on save is not working with Prettier formatter

Reply:
Yes 🤔

Reply:
done and somehow it was deactivated